### PR TITLE
stack810.yaml: bump

### PR DESCRIPTION
**Pull request by domenkozar**
Needs `diagrams-svg` release.

### CODE REVIEW

File: stack810.yaml [hunk 1]
```diff
@@ -1,6 +1,4 @@
-resolver: nightly-2020-02-13
-compiler: ghc-8.10.1
-allow-newer: true
```
Comment (cocreature) [inline stack810.yaml:3]: Can we just keep the `allow-newer` for now? Clearly `diagrams-svg` needs it.

**Comment by pepeiborra** [comment]
It might be a good idea to make `benchHist` non buildable by default, or turn it into a benchmark. I believe @alanz did this for the hls fork of ghcide

**Comment by domenkozar** [comment]
It builds now but benchmarking has a lot of these:

```
Warning: Failed to decode module interface: /home/ielectric/dev/ghcide/.stack-work/dist/x86_64-linux-nix/Cabal-3.2.0.0/build/ghcide/ghcide-tmp/Utils.hi Decoding failure: Invalid magic: e49ceb0f
```

**Comment by cocreature** [comment]
> It builds now but benchmarking has a lot of these:

That’s a stack bug https://github.com/commercialhaskell/stack/issues/5134

**Pull request merged**